portal/package.json

```diff
@@ -3,6 +3,7 @@
 		"@mysten/bcs": "^1.0.2",
 		"@mysten/sui": "^1.1.2",
 		"base-x": "^4.0.0",
+		"parse-domain": "^8.0.2",
```

Comment:
Additional Remark: [This notice](https://github.com/peerigon/parse-domain?tab=readme-ov-file#updates) on the package seems very important. Do we have an automated way to update this "everytime you start or build your application"?

Reply:
That's true, I should include `npx parse-domain-update` to the build and test steps.

Reply:
Quoting parse-domain [README](https://github.com/peerigon/parse-domain#readme):

>This package comes with a prebuilt list that has been downloaded at the time of npm publish. 

Therefore, if we use a fixed version of the `parse-domain`  (e.g. `8.0.2`, the current one) the code will still be functional even if we do add the walrus.site to the suffix list.

Reply:
Perfect!